docs/src/main/sphinx/connector/metastores.md

```diff
@@ -183,25 +183,23 @@ properties:
 * - `hive.metastore.thrift.txn-lock-max-wait`
   - Maximum time to wait to acquire hive transaction lock.
   - `10m`
-* - ``hive.metastore.http.client.bearer-token``
-  - Bearer token used to authenticate with the metastore service
-    when https transport mode is used. This must not be set when
-    using http url.
+* - `hive.metastore.http.client.bearer-token`
+  - Bearer token to use for authentication with the metastore service when HTTPS
+    transport mode is used. This must not be set with an HTTP URL.
```

Comment:
Question here.. where would that mode or URI or URL or whatever be set

Reply:
with hive.metastore.thrift.client.ssl.enabled ? .. an then in either case.. its not really HTTPS vs HTTP .. its about TLS enabled with thrift:// or not right? Or can we consider that to be equivalent?

Reply:
This mode is used when the `hive.metastore.uri` configuration uses a http or https scheme in the url.